### DISCUSSION TITLE
Nuget publish for GDAL 3.6.0

### DISCUSSION THREAD

**Discussion by bjornharrtell** [MaxRev-Dev/gdal.netcore#82] [General]
Hi @MaxRev-Dev if you got the time I'd highly appreciate if you are able to make an update to the nuget packages to align with GDAL 3.6.0

**Comment by MaxRev-Dev** [comment]
Hey @bjornharrtell. Happy to announce, I'm already working on this, and I hope to ship it within this or next week. 
I had some troubles on building 3.5 as it's partially on CMake, so I've jumped to 3.6 and have successfully built two runtimes. As GDAL itself moved to PowerShell and CMake scripting, the same happens to **gdal.netcore**. Now I'm updating the packaging system - binary collectors and library managing scripts to provide a solid package for both runtimes.

**Comment by sirhcybe** [comment]
I just found an issue I have been debugging was a bug in GDAL 3.3.0 and is fixed in 3.6.0. So it's nice to come here to see about and upgrade and find that it's already in the works. I don't want to step on anyone's toes but if there's anything I can do to help, please let me know.

**Comment by MaxRev-Dev** [comment]
Finally done with testing and publishing packages. Added windows matrix to CI testing. 
Everyone, please let me know about any issues in this release so the patches can be created.
As this build was created under the base image of bullseye (Debian 11), not sure will it work on CentOS7 or other distros.
Windows should not have any issues, though. 
And I think that after the New Year we can have an update on storing/loading binaries from specific location (#21, #9).
Please, tell me what you think!
Best wishes, Max

**Reply by bertt** [reply]
did a quick test, works great! Just observed driver 'MySQL' is included on Windows but not on Linux is this intentional? Got the list of drivers from Ogr.GetDriverCount()  / Ogr.GetDriver(i).GetName().

**Reply by MaxRev-Dev** [reply]
Good observation. No, this was not intentional. During the compilation, some of the dependencies on the linux side were not installed as they require a lot of boost dependencies (more than 80). And there was a collision with dependencies of version that was installed from apt manager. This will be fixed by a patch on the linux side, adding that driver to the package.

**Reply by sirhcybe** [reply]
I also tested this and it resolved my WKT parsing bug. Thanks so much!
